### PR TITLE
Only provide legacy styles for legacy interactives

### PR DESCRIPTION
**Pull request by OllysCoding**
## What does this change?

This PR depends on a PR from frontend: https://github.com/guardian/frontend/pull/23923, it should not be merged until after the frontend PR is merged.

Updates interactives so the legacy 'global' styles are only applied if the interactive came out _before_ the switchover date.

## Why?

We don't really want to be supporting these legacy global styles going forward on new interactives.

### CODE REVIEW

File: src/web/layouts/InteractiveImmersiveLayout.tsx [hunk 1]
```diff
@@ -196,7 +196,9 @@ export const InteractiveImmersiveLayout = ({
 
 	return (
 		<>
-			<Global styles={interactiveGlobalStyles} />
+			{CAPI.isLegacyInteractive && (
+				<Global styles={interactiveGlobalStyles} />
```
Comment (jlieb10) [inline src/web/layouts/InteractiveImmersiveLayout.tsx:200]: I wonder if renaming this to `interactiveLegacyStyles` would be helpful, too.

**Review by jlieb10 — APPROVED** [review]
Looks good 👍  Just have a think about my rename question

**Review by shtukas — APPROVED** [review]

File: src/web/layouts/InteractiveImmersiveLayout.tsx (hunk 1 above)
Comment (OllysCoding) [reply src/web/layouts/InteractiveImmersiveLayout.tsx:200]: Good shout

Comment (OllysCoding) [reply src/web/layouts/InteractiveImmersiveLayout.tsx]: Done :D

**Comment by OllysCoding** [comment]
Closing

**Pull request closed without merging**